actions/run-nextstrain-build-on-aws-batch/action.yaml

```diff
@@ -0,0 +1,68 @@
+name: Run Nextstrain build on AWS Batch
+description: >-
+  This GitHub Actions action is intended to be called by workflows in our other
+  repos when they need to run a nextstrain build on AWS Batch.
+
+inputs:
+  runtime-envvar-names:
+    description: >-
+      A list of environment variable names for environment variables that
```

Comment:
Where do these come from? Must they be a subset of the env variables set in the `deploy-build-to-aws-batch` below, or can you also supply your own when calling the rule?

Reply:
They can be _any_ environment variable you have set in the workflow when you are using this custom action. The `deploy-build-to-aws-batch` below is just an example template. In the [trial workflow](https://github.com/nextstrain/seasonal-flu/actions/runs/4897723596/workflow) I did for the seasonal-flu build, all three levels of env (workflow, job, step) were able to be passed to the build.